### ISSUE TITLE
ModuleNotFoundError: No module named 'six'

### ISSUE THREAD

**Issue by nesl247** [jshimko/kube-tools-aws#3]
Getting `ModuleNotFoundError: No module named 'six'` when trying to run `aws eks`.

**Comment by nesl247** [comment]
I discovered this is because you cleanup `py-pip` but `six` is a dependency needed by `aws`.

**Comment by jshimko** [comment]
Hi @nesl247.   While I use this container nearly every day, I haven't run into this issue and I have limited free time to debug issues for other folks (although it's great to hear other people find value in this project!).  If you could provide some basic reproduction steps for this issue, I'm happy to take a look and fix it.  So just a few more details about the commands you're trying to run would be great.  I assume it's an easy fix in the install steps, but in the interest of time, I don't want to have to guess the ways other people are using this in their workflows.  Any extra info is greatly appreciated!

**Comment by nesl247** [comment]
Sure. `docker run --rm -it jshimko/kube-tools-aws sh` and then `aws eks`. The issue is as mentioned, that `aws` relies upon `py-six` being installed, but it gets removed when removing `py-pip`.

```
❯ docker run --rm -it jshimko/kube-tools-aws sh
/ # aws
Traceback (most recent call last):
  File "/usr/bin/aws", line 19, in <module>
    import awscli.clidriver
  File "/usr/lib/python3.8/site-packages/awscli/clidriver.py", line 17, in <module>
    import botocore.session
  File "/usr/lib/python3.8/site-packages/botocore/session.py", line 29, in <module>
    import botocore.configloader
  File "/usr/lib/python3.8/site-packages/botocore/configloader.py", line 19, in <module>
    from botocore.compat import six
  File "/usr/lib/python3.8/site-packages/botocore/compat.py", line 26, in <module>
    from dateutil.tz import tzlocal
  File "/usr/lib/python3.8/site-packages/dateutil/tz/__init__.py", line 2, in <module>
    from .tz import *
  File "/usr/lib/python3.8/site-packages/dateutil/tz/tz.py", line 19, in <module>
    import six
ModuleNotFoundError: No module named 'six'
/ # apt add --update py-pip
sh: apt: not found
/ # apk add --update py-pip
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz
(1/26) Installing py3-appdirs (1.4.4-r1)
(2/26) Installing py3-ordered-set (4.0.1-r0)
(3/26) Installing py3-parsing (2.4.7-r0)
(4/26) Installing py3-six (1.15.0-r0)
(5/26) Installing py3-packaging (20.4-r0)
(6/26) Installing py3-setuptools (47.0.0-r0)
(7/26) Installing py3-chardet (3.0.4-r4)
(8/26) Installing py3-idna (2.9-r0)
(9/26) Installing py3-certifi (2020.4.5.1-r0)
(10/26) Installing py3-urllib3 (1.25.9-r0)
(11/26) Installing py3-requests (2.23.0-r0)
(12/26) Installing py3-msgpack (1.0.0-r0)
(13/26) Installing py3-lockfile (0.12.2-r3)
(14/26) Installing py3-cachecontrol (0.12.6-r0)
(15/26) Installing py3-colorama (0.4.3-r0)
(16/26) Installing py3-distlib (0.3.0-r0)
(17/26) Installing py3-distro (1.5.0-r1)
(18/26) Installing py3-webencodings (0.5.1-r3)
(19/26) Installing py3-html5lib (1.0.1-r4)
(20/26) Installing py3-pytoml (0.1.21-r0)
(21/26) Installing py3-pep517 (0.8.2-r0)
(22/26) Installing py3-progress (1.5-r0)
(23/26) Installing py3-toml (0.10.1-r0)
(24/26) Installing py3-retrying (1.3.3-r0)
(25/26) Installing py3-contextlib2 (0.6.0-r0)
(26/26) Installing py3-pip (20.1.1-r0)
Executing busybox-1.31.1-r19.trigger
OK: 84 MiB in 58 packages
```

**Comment by jshimko** [comment]
Ok, should be good to go with 3.5.0.

`docker run --rm -it jshimko/kube-tools-aws:3.5.0 aws eks help`

**Comment by jshimko** [comment]
Closing.  This was fixed back in 3.5.0.
